packages/runtime/src/next/client.ts

```diff
@@ -404,14 +404,20 @@ export class Makeswift {
 
   async getPageSnapshot(
     pathname: string,
-    { preview: previewOverride = false }: { preview?: boolean } = {},
+    { preview: previewOverride = false, locale }: { preview?: boolean; locale?: string } = {},
```

Comment:
Let's make this `unstable_locale` until we're ready for a stable release of our localization features

Reply:
True. Updated.